Kitodo/src/main/webapp/WEB-INF/templates/includes/metadataEditor/logicalStructure.xhtml

```diff
@@ -232,6 +240,20 @@
                                 paginationForm:paginationWrapperPanel
                                 metadataAccordion:logicalMetadataWrapperPanel
                                 galleryWrapperPanel"/>
+            <p:menuitem value="#{msgs.deleteMediaPartial}"
+                        rendered="#{SecurityAccessController.hasAuthorityToDeleteMedia() and DataEditorForm.structurePanel.selectedLogicalNode.type eq StructurePanel.MEDIA_PARTIAL_NODE_TYPE}"
```

Comment:
```suggestion
                        rendered="#{SecurityAccessController.hasAuthorityToDeleteMedia() and DataEditorForm.structurePanel.selectedLogicalNode.type eq DataEditorForm.structurePanel.MEDIA_PARTIAL_NODE_TYPE}"
```
The class `StructurePanel` is not visible here, so I think we need to retrieve it via the `DataEditorForm`, don't we?  (same above)

Reply:
I get following exceptions when i change that to DataEditorForm.structurePanel. 

![image](https://github.com/kitodo/kitodo-production/assets/3832618/dc8201d6-83a2-4649-ae82-cf3ed55ee241)

In this file, other calls to class constants to structure panel are done in the same way as I do. 
https://github.com/kitodo/kitodo-production/blob/2960f1fd0f425170fb6ac5c7fb544c1c6e626955/Kitodo/src/main/webapp/WEB-INF/templates/includes/metadataEditor/logicalStructure.xhtml#L115

https://github.com/kitodo/kitodo-production/blob/2960f1fd0f425170fb6ac5c7fb544c1c6e626955/Kitodo/src/main/webapp/WEB-INF/templates/includes/metadataEditor/logicalStructure.xhtml#L133